src/store/getUrlData.ts

```diff
@@ -1,33 +1,49 @@
+import {normalizePathSlashes} from '../utils';
+
 export const getUrlData = ({
-    href,
     singleClusterMode,
     customBackend,
 }: {
-    href: string;
     singleClusterMode: boolean;
     customBackend?: string;
 }) => {
+    // UI could be located in "monitoring" or "ui" folders
```

Comment:
could you please provide more verbose comment what "UI located" means here

Reply:
Added example